README.md

```diff
@@ -147,7 +163,7 @@ Prefixes and suffixes are used for prepending or appending static text to a form
 </div>
 ```
 
-[Prefixes and suffixes examples](https://origami-build.ft.com/v2/files/o-forms@%5E1/demos/prefix-suffix.html)
+[Prefixes and suffixes examples](https://www.ft.com/__origami/service/build/v2/demos/o-forms/prefix-suffix)
 
 #### "unskin" a form element
```

Comment:
not needed for this PR but does unskinning work with this toggle? If not can you add an issue as a reminder to fix it later?

Reply:
Oh I didn't see that. It does not. https://github.com/Financial-Times/o-forms/issues/156